admin/data/parcelpro-shipping-settings-fields.php

```diff
@@ -125,6 +125,34 @@
         'description' => 'Customer note (billing address) invullen bij het veld "inhoud" in het verzendsysteen.',
         'desc_tip'    => true
     ),
+    'parcelpro_forecast_settings_postnl_show_delivery_date' => array(
+        'type'     => 'checkbox',
+        'title'    => 'Verwachte levertijd inschakelen (PostNl)',
+        'label'    => 'Verwachte levertijd inschakelen',
+        'desc_tip' => true,
+        'default'  => 'no',
+    ),
+    'parcelpro_forecast_settings_postnl_cutoff_time' => array(
```

Comment:
Ik weet niet of "cutoff time" de meest logische/begrijpbare term is voor mensen die dit moeten configureren. Mogelijk dat iets als "laatste verzendtijd" een duidelijkere term is.

Verder zou ik hier graag een tooltip voor zien met wat extra info. In de Magento module staat er dit in de tooltip:

> De laatste tijd waarop bestellingen verzonden kunnen worden. Na deze tijd wordt de verwachte bezorging vanaf de volgende dag genomen. Bijvoorbeeld: "22:00 Europe/Amsterdam".

Zoiets lijkt me hier ook goed om toe te voegen, om zo uit te leggen wat de functie doet en een voorbeeld te geven.

Reply:
Is het intentional dat je de title/label waarden hetzelfde hebt gelaten @dutchmartin ? Voelt nogal redundant om daar 2 keer dezelfde tekst te hebben staan.

Reply:
Ik heb dat nu aangepast in de tooltip en een placeholder erop gezet